.gitignore

```diff
@@ -1,5 +1,7 @@
 .vscode/
-wurst/war3map.j
-objectEditingOutput/*
-compiled.j.txt
+/_build
 
+*.w3x
+*.w3m
+
+wurst.dependencies
```

Comment:
What impact does ignoring + removing this file have?

Reply:
It's local data that contains an absolute path specific to your system. Local data shouldn't be committed to repos. Also it is generated. What impact does having this file in the repo do, except misleading and possibly throwing errors of unresolvable paths?

Reply:
I don't know what impact having it has, but I do know that:

- if I clone the repo, open vscode, and hit run map, it will work
- This has always worked, so my bias is for what I know doesn't not work.

I understand what the desired outcome is from your side. But, my goal is to understand how users and developers will be impacted by this change.

Can you answer, specifically, what processes will generate this file? Is it wurst setup when you hit "update"? Or maybe the vscode plugin will refresh this?